### PR TITLE
More small website changes:

### PR DESCRIPTION
**Pull request by srcarter3**
- the alttext for images still didn't seem to appear in a google search, so added a little more actual text describing the errors encountered, with hopes that google will find this page

### CODE REVIEW

File: docs/install/common-problems.md [hunk 1]
```diff
@@ -41,7 +41,10 @@ If that is not an option, then the only remote access we recommend is using some
 
 One common error some users are seeing manifests itself just after selecting an EDEX server to connect to.  The following error dialogs may show up:
 
+- *Error purging logs*  
 ![Error purging logs. Reason: Error occurred during purge and rotate](../images/errorPurgingLogs.png)
+
+- *Error instantiating workbench*
```
Comment (tiffanycmeyer13) [inline docs/install/common-problems.md]: Want to add  ": null" or just leave it?

Comment (srcarter3) [reply docs/install/common-problems.md]: I thought about it but couldn't decide... what do you think?

Comment (tiffanycmeyer13) [reply docs/install/common-problems.md]: I think for that one it should have it

Comment (srcarter3) [reply docs/install/common-problems.md]: Okay just added that!

**Pull request merged**